### PR TITLE
use video duration from events

### PR DESCRIPTION
**Pull request by muhammad-ammar**
[EDUCATOR-1411](https://openedx.atlassian.net/browse/EDUCATOR-1411)

### CODE REVIEW

File: edx/analytics/tasks/insights/video.py [hunk 1]
```diff
@@ -326,18 +328,17 @@ def reducer(self, key, events):
...
-            timestamp, event_type, current_time, old_time, youtube_id = event
+            timestamp, event_type, current_time, old_time, youtube_id, video_duration = event
             parsed_timestamp = ciso8601.parse_datetime(timestamp)
             if current_time is not None:
                 current_time = float(current_time)
             if old_time is not None:
                 old_time = float(old_time)
 
-            def start_viewing():
+            def start_viewing(video_duration):
```
Comment (brianhw) [inline edx/analytics/tasks/insights/video.py]: I don't think this is needed -- video_duration is defined now on 331, so it's already in context.  For example, youtube_id is not passed into start_viewing(), nor is current_time and a raft of others.

File: edx/analytics/tasks/insights/video.py [hunk 2]
```diff
@@ -408,7 +409,7 @@ def end_viewing(end_time):
                     ## self.incr_counter(self.counter_category_name, 'Discard Viewing Start', 1)
                     ## self.incr_counter(self.counter_category_name, 'Discard Viewing', 1)
                     pass
-                viewing = start_viewing()
+                viewing = start_viewing(video_duration)
```
Comment (brianhw) [inline edx/analytics/tasks/insights/video.py]: Again, I don't think this change is needed.

**Review by brianhw — COMMENTED** [review]
Small nits.  Otherwise looking good.

**Comment by muhammad-ammar** [comment]
@brianhw Addressed your feedback. Video acceptance test is fixed and [build is passing](http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/1415/console). Is this good to merge?

**Comment by muhammad-ammar** [comment]
@edx/analytics This needs your attention. Please have a look.

**Review by brianhw — APPROVED** [review]
LGTM. 👍

**Review by HassanJaveed84 — APPROVED** [review]
LGTM 👍

**Comment by muhammad-ammar** [comment]
@HassanJaveed84 Thanks a lot for all the help

**Pull request merged**